pilot/pkg/networking/core/v1alpha3/route/route.go

```diff
@@ -845,7 +851,9 @@ func BuildDefaultHTTPInboundRoute(node *model.Proxy, clusterName string, operati
 			Route: &route.RouteAction{
 				ClusterSpecifier: &route.RouteAction_Cluster{Cluster: clusterName},
 				Timeout:          notimeout,
-				MaxGrpcTimeout:   notimeout,
+				MaxStreamDuration: &route.RouteAction_MaxStreamDuration{
```

Comment:
same question here: when was this implemented? Would this break old proxies?

Reply:
Yeah, I am not sure. I copied the change from another go control plane update PR. I reverted the change and add no lint directive.